OSTools.py

```diff
@@ -159,5 +164,12 @@ def warn(msg):
             while Button.CENTER in charlie.buttons.pressed():
                 pass
             logMsg = 1
-
-            
\ No newline at end of file
+    
+    def info(self, msg):
+        ''' Makes a log output, without showing anything on the EV3 screen'''
+        global logMsg
+        sound(SoundFile.GENERAL_ALERT)
+        if True:
+            timeStamp = strftime("%d.%m.%Y %H:%M:%S")
+            print("[" + timeStamp + " Info]", exception, msg)
+    
```

Comment:
please put all the sound allerts into the if statemnts for the screen printing...

Reply:
Sorry, I don't get what you mean?

Reply:
the sound(SoundFile.GENERAL_ALERT) into the if statement in which the messages are printed on the screen

Reply:
Gonna close this now, if you don't react in a timely manner.